helm-chart/renku/values.yaml

```diff
@@ -1613,6 +1613,21 @@ dataService:
   affinity: {}
   # This defines the maximum number of pinned projects in user preferences
   maxPinnedProjects: 5
+
+authz:
+  replicaCount: 1
+  image:
+    repository: authzed/spicedb
+    pullPolicy: IfNotPresent
+    tag: "v1.30.0"
+  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 10
```

Comment:
This seems quite high. Has this been necessary during testing?

Reply:
I have not been able to do any load testing. It is still too early for this because the authorization service is not hooked up to anything in the backend yet. So I cannot do any meaningful load tests. I changed this to 3 and we can revise it further.

But this is also very dependent on each deployment. If the deployment serves 10 people it will need less replicas than something like renkulab.io.

Either way I can do some tests when I implement stuff on the backend and revise this. The autoscaling is also off by default.